fury/ui.py

```diff
@@ -1071,9 +1071,9 @@ class TextBlock2D(UI):
         Adds text shadow.
     """
 
-    def __init__(self, text="Text Block", font_size=18, font_family='Arial',
+    def __init__(self, text="Text Block", font_size=None, font_family='Arial',
                  justification='left', vertical_justification="bottom",
-                 bold=False, italic=False, shadow=False,
+                 bold=False, italic=False, shadow=False, position2=None,
```

Comment:
`position2` should not be a parameter. You should add a parameter `size` to respect the API with the other UI.  With position and size, you can compute and update `position2`.

Reply:
HI @skoudoro , got your point. In that case I have to add a `@size.setter` method right?